docs/developer-guide/README.md

```diff
@@ -1,9 +1,31 @@
 # Developer Guide
 
-Welcome to the luma.gl developer guide.
+To get started developing luma.gl, first make sure to install all dependancies from the repository root:
```

Comment:
This entire page is great, but `docs/developer-guide` is supposed to be a "luma.gl user" Developer Guide, however `yarn bootstrap` is information that belongs in a "luma.gl developer" Developer Guide.

95% of all users will only install and import luma.gl, never clone the repo and never need to run `yarn bootstrap`, so having this information here risks confusing users.

I agree that we should have information about how to build and develop luma.gl itself but it should be a separate page (it could go in the developer guide, but at the very least it should not be the main README/first page of the Developer Guide).

Reply:
Good call, there's a separate section for developing luma itself. I'll move this stuff there.